image/tools/lib/component/resources.sh

```diff
@@ -90,7 +90,7 @@ function component_dump_data {
     local dest=$1
     local namespaces=$(get_middleware_namespaces)
 
-    for ns in ${namespaces}; do
+    for ns in ${namespaces} "default"; do
```

Comment:
Does it make sense to use this var instead of hardcoding it: https://github.com/integr8ly/backup-container-image/blob/master/templates/openshift/backup-cronjob-template.yaml#L71?

Reply:
I don't think so. That flag is only used for sso to allow the component secret to be in a different namespace than `default`. Usually this namespace would be one that is already included in the middleware namespace so it should also not be added additionally.